### ISSUE TITLE
build errors

### ISSUE THREAD

**Issue by xpech** [ycaseau/CLAIRE4#3]
Hi Yves,

I guess the core.go and code-meta.go are redundant ? the *-meta seems to be the good one.
I've created a pull request.

```
xpech@XavBook src % go build claire4.go
# Core
/opt/local/lib/go/src/Core/Core.go:11:6: GeneralError redeclared in this block
	/opt/local/lib/go/src/Core/Core-meta.go:11:6: previous declaration
....

```

**Comment by ycaseau** [comment]
Absolutely,  the <module>.go files have been renamed into <module>-meta.go.
The presence of Core.go or others is a git synchronization error which I have now fixed.

**Comment by ycaseau** [comment]
The old <module>.go files (Core, Reader, Language ...) have re-appeared. this should be fixed definitely with 4.1.14 which I am now also testing on Ubuntu (using an AWS instance)

**Comment by ycaseau** [comment]
this is closed
